README.md

```diff
@@ -6,6 +6,18 @@
 
 Head over to the [generated docs](https://owncloud.github.io/owncloud-design-system/) for more information!
 
+## Announcement: This is (likely) not the repo you're looking for
+The design system is alive and kicking. We've decided to move the code into the [ownCloud Web](https://github.com/owncloud/web) mono-repository to streamline
+development processes. Publishing to [npmjs](https://npmjs.com) and deploying the [docs](https://owncloud.design) will happen from 
+the web mono-repo as well in the near future. This means:
+- If you're a developer using the design system, nothing changes for you.
+- If you want to contribute to the design system, please open a pull request in [ownCloud Web](https://github.com/owncloud/web).
+
+As the actively supported [web stable-6.0 branch](https://github.com/owncloud/web/tree/stable-6.0) depends on the design system version `14.0.x`,
+we'll keep the [design system stable-14.0 branch](https://github.com/owncloud/owncloud-design-system/tree/stable-14.0) in this repo alive as long as
+web v6.0 is supported. As soon as web v6.0 is not supported anymore, we'll archive the design system repository.
+(side note: end of support for v6.0 is not known, yet. We'll update this document as soon as it's decided.)
```

Comment:
I think I would move this whole block at the very top to make it even more prominent. 

Not sure about that, but maybe you could link to the [correct path inside the repo](https://github.com/owncloud/web/tree/master/packages/design-system)

Reply:
Moved to the top 👍 

I added the link to the path to the sentence where I mentioned contributing to the design system. Ok or do you have a better place?

Reply:
Perfect, thanks!